[PT2][partitioners] Add aten.split to view_ops list

ShatianWang opened the pull request:
Summary: Add `aten.split` to view_ops list in partitioners.py

Test Plan:
na

Rollback Plan:

Differential Revision: D76011951




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben

ShatianWang commented:
Re-trigger EasyCLA check /easycla

xuanzhang816 reviewed (approved):
I think once we fix the test failures, we are good to go.

ShatianWang committed:
[PT2][partitioners] Add aten.split to view_ops list (#155424)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/155424

Add `aten.split` to view_ops list in partitioners.py

Test Plan:
na

Rollback Plan:

Reviewed By: xuanzhang816

Differential Revision: D76011951

clee2000 commented:
The test stopped failing stopped after the revert, I added `ciflow/slow` to run the test on the PR

xuanzhang816 commented:
Unable to reproduce the unittest error locally after rebasing on main. 

Relanding through -- https://github.com/pytorch/pytorch/pull/155943
